db-init.sql

```diff
@@ -0,0 +1,166 @@
+-- PersonCategories
+	-- all the different types a person can be
+
+CREATE TABLE PersonCategories (
+	personCategoryId SERIAL PRIMARY KEY NOT NULL,
+	personCategoryName VARCHAR(255) NOT NULL,
+);
+
+	CREATE UNIQUE INDEX personCategoryName_lower ON PersonCategories(lower(personCategoryName));
+
+	INSERT INTO PersonCategories VALUES (0, 'farmers');
+	INSERT INTO PersonCategories VALUES (1, 'traders');
+	INSERT INTO PersonCategories VALUES (2, 'admins');
+	INSERT INTO PersonCategories VALUES (3, 'monitor');
+
+-- PersonAttributeTypes
+	--  all the different types of attributes a person can have
+
+CREATE TABLE PersonAttributeTypes (
+	attrId SERIAL PRIMARY KEY NOT NULL,
+	attrName VARCHAR(255) NOT NULL
+);
+
+	CREATE UNIQUE INDEX personAttrName_lower ON PersonAttributeTypes(lower(attrName));
```

Comment:
out of curiosity, is it good practice to make these indexes lower case? will we ever have an attribute type or something that is an abbreviation for something and should be upper case?

Reply:
also, does this mean that "passwordHash" will become "passwordhash"?